lib/linguist/languages.yml

```diff
@@ -7829,12 +7829,14 @@ jq:
   language_id: 905371884
 just:
   type: programming
-  aliases:
-  - Justfile
   color: "#384d54"
   tm_scope: source.just
   filenames:
+  - JUSTFILE
   - Justfile
+  - justfile
+  extensions:
+  - ".just"
```

Comment:
> Add the `.just` extension, which is in common use. See this search: https://github.com/search?q=extension%3Ajust

Using GitHub's new Code Search engine, I only found 88 results for [`path:**/?*.just`](https://github.com/search?q=path%3A**%2F%3F*.just&type=code), which falls far short of [our real-world usage criteria](https://github.com/github/linguist/blob/59b2d88b2242e6062384e5fb876668cc30ead951/CONTRIBUTING.md#adding-an-extension-to-a-language) (at least 200 unique users/repos that use the new extension).

Reply:
Done! I removed the `.just` extension.

Is there a way for me to access the new Code search engine? The link `https://github.com/search?q=path%3A**%2F%3F*.just&type=code` doesn't show any results.

Reply:
Yup: https://github.com/features/code-search/